rllib/algorithms/dqn/tests/test_dqn.py

```diff
@@ -26,7 +27,9 @@ def test_dqn_compilation(self):
         num_iterations = 1
         config = dqn.dqn.DQNConfig().rollouts(num_rollout_workers=2)
 
-        for _ in framework_iterator(config, with_eager_tracing=True):
+        for _ in framework_iterator(
+            config, frameworks="torch"
+        ):  # with_eager_tracing=True):
```

Comment:
Should revert to original to test all frameworks

Reply:
great catch!
done